app/controllers/application_controller.rb

```diff
@@ -471,11 +471,23 @@ def fetch_translations
     WebTranslateIt.fetch_translations
   end
 
+  def redirect_https?
+    always_use_ssl = APP_CONFIG.always_use_ssl.to_s.downcase == "true"
+    return !request.ssl? && always_use_ssl
+  end
+
+  def redirect_https
+    redirect_to protocol: "https://", status: :moved_permanently if redirect_https?
```

Comment:
`if redirect_https?` is redundant here. This has been already checked in the `check_http_auth` method. So it can be removed.

Again, might be a matter of taste, but I'd maybe add an exclamation mark to the method name (`redirect_https!`) to emphasize that the redirect is _really_ happening.